core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -399,6 +401,9 @@ public static <E extends Entry<Key,Value>> TabletMetadata convertRow(Iterator<E>
           extCompBuilder.put(ExternalCompactionId.of(qual),
               ExternalCompactionMetadata.fromJson(val));
           break;
+        case ChoppedColumnFamily.STR_NAME:
+          te.chopped = true;
```

Comment:
Could do this if don't want to add a method to get chopped.

```suggestion
```

Reply:
I don't follow your suggestion. It looks like you are just deleting the line?

Reply:
I am guessing you are saying to have a getter and setter, so we can check the fetched list.

Reply:
I Was just thinking that just doing nothing when seeing chopped would be ok if the getter is not added now.  Nothing would be calling the getter.